draft-ietf-pquip-pqc-engineers.md

```diff
@@ -225,7 +225,9 @@ Any asymmetric cryptographic algorithm based on integer factorization, finite fi
 
 # Invariants of Post-Quantum Cryptography
 
- In the context of PQC, symmetric-key cryptographic algorithms are generally not directly impacted by quantum computing advancements. Symmetric-key cryptography, which includes keyed primitives such as block ciphers (e.g., AES) and message authentication mechanisms (e.g., HMAC-SHA2), rely on secret keys shared between the sender and receiver. Symmetric cryptography also includes hash functions (e.g., SHA-256) that are used for secure message digesting without any shared key material. HMAC is a specific construction that utilizes a cryptographic hash function (such as SHA-2) and a secret key shared between the sender and receiver to produce a message authentication code. CRQCs, in theory, do not offer substantial advantages in breaking symmetric-key algorithms compared to classical computers (see {{symmetric}} for more details).
+ In the context of PQC, symmetric-key cryptographic algorithms are generally not directly impacted by quantum computing advancements. Symmetric-key cryptography, which includes keyed primitives such as block ciphers (e.g., AES) and message authentication mechanisms (e.g., HMAC-SHA2), rely on secret keys shared between the sender and receiver. Symmetric cryptography also includes hash functions (e.g., SHA-256) that are used for secure message digesting without any shared key material. HMAC is a specific construction that utilizes a cryptographic hash function (such as SHA-2) and a secret key shared between the sender and receiver to produce a message authentication code. 
+ 
+CRQCs, in theory, do not offer substantial advantages in breaking symmetric-key algorithms compared to classical computers meaning that current symmetric algorithms can continue to be used with potentially small increases to key size to stay ahead of quantum-boosted brute-forcing attacks (see {{symmetric}} for more details).
 
```

Comment:
I would not say "small increases to key size"; it may require doubling the key size (128 to 256-bit).

Reply:
Yeah, it's a matter of perspective. The additional 16 bytes and additional CPU time is trivial in the majority of situations, so it's tempting to characterize the change as "small". But if you have a fixed size database field or limited secure memory, the effort to make this "small" change may be "large".

Perhaps "straightforward" is the right adjective, because that's why not a lot of attention is being paid here. There probably are no changes at all needed to the vast majority of symmetric systems, and if a change is needed, it's fairly straightforward, even if deploying it in realistic scenarios may not always be.

Reply:
Yes, proposed change works for me.

CRQCs, in theory, do not offer substantial advantages in breaking symmetric-key algorithms compared to classical computers, meaning that current symmetric algorithms can continue to be used with potentially straightforward increases to key size to stay ahead of quantum-boosted brute-forcing attacks (see {{symmetric}} for more details).